src/Glpi/Marketplace/View.php

```diff
@@ -884,19 +884,22 @@ public static function getButtons(string $plugin_key = ""): string
                 }
             }
 
-            $buttons .= TemplateRenderer::getInstance()->render('components/plugin_uninstall_modal.html.twig', [
-                'plugin_name' => $plugin_inst->getField('name'),
+            $buttons .= TemplateRenderer::getInstance()->render('components/danger_modal.html.twig', [
                 'modal_id' => 'uninstallModal' . $plugin_inst->getField('directory'),
                 'open_btn' => '<button data-bs-toggle="modal"
                                        data-bs-target="#uninstallModal' . $plugin_inst->getField('directory') . '"
                                        title="' . __s('Uninstall') . '">
                                    <i class="ti ti-folder-x"></i>
                                </button>',
-                'uninstall_btn' => '<a href="#" class="btn btn-danger w-100 modify_plugin"
+                'confirm_btn' => '<a href="#" class="btn btn-danger w-100 modify_plugin"
                                        data-action="uninstall_plugin"
                                        data-bs-dismiss="modal">
                                        ' . _x("button", "Uninstall") . '
                                    </a>',
+                'content' => sprintf(
+                    __('By uninstalling the "%s" plugin you will lose all the data of the plugin.'),
+                    $plugin_inst->getField('name')
```

Comment:
`content` will not be escaped anymore due to the usage of the `|raw` filter
```suggestion
                                __s('By uninstalling the "%s" plugin you will lose all the data of the plugin.'),
                                htmlspecialchars($plugin_inst->getField('name'))
```